docs/notebooks/cli.ipynb

```diff
@@ -262,7 +262,11 @@
    "source": [
     "# Create an output file with just the flag value and no aggregation (dims=None)\n",
     "\n",
-    "!xclim -i example_data.nc -o flag_output.nc dataflags -d none"
+    "!xclim -i example_data.nc -o flag_output.nc dataflags -d none\n",
+    "\n",
+    "# Need to wait until the file is written\n",
+    "\n",
+    "!sleep 2s"
```

Comment:
Does the command returns before the file is written? Weird!?

Reply:
It was happening to me, so I added it in case. Weird behaviour for sure.